src/main/java/com/neuvector/ScannerEnvironmentImpl.java

```diff
@@ -0,0 +1,22 @@
+package com.neuvector;
+
+public class ScannerEnvironmentImpl implements ScannerEnvironment {
+
+    private String containerSocket;
+
+    @Override
+    public void create() {
+        String containerRuntimeSocket = System.getenv(ScannerEnvironment.ENV_CONTAINER_RUNTIME_SOCKET);
```

Comment:
We should not use System.getenv but rather inject this value from insight-scanner as we do for similar values. System.getenv does not work in Jenkins.